src/components/AddItem.js

```diff
@@ -1,34 +1,78 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { withFirestore } from 'react-firestore';
+import '../styles/AddItem.css';
+import { v4 as uuidv4 } from 'uuid';
 
 const AddItem = props => {
+  const emptyShoppingItem = {
+    name: '',
+    lastPurchasedDate: null,
```

Comment:
I was wondering if we need to save `lastPurchasedDate` attribute in the database? If yes, I think we are missing this attribute between lines 26-30, while adding the object to the database.

Reply:
That's a good point. We weren't sure about lastPurchaseDate. Let me post it in slack.

Reply:
Yep! LastPurchaseDate should just be today's date. 👍

Reply:
Yep! LastPurchaseDate should just be today's date. 👍 Once you stick that date in here, we should be good to merge!